src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,52 @@
+package duke.task;
+
```

Comment:
You might want to standardise across files if a line is left in between package and the first import statement or not!